.circleci/packages/config.yaml

```diff
@@ -0,0 +1,54 @@
+
+version:
+  release:
+    match: '^v[0-9]+.[0-9]+.[0-9]+'
+    value: '{{env.CIRCLE_TAG[1:]}}'
+  alpha:
+    match: '^v3.0.0.alpha.[0-9]+.[0-9]+'
```

Comment:
This should be `^v3.0.0-0.alpha.[0-9]+.[0-9]+` (the `-0` is important for package managers).